src/sinks/influxdb/metrics.rs

```diff
@@ -128,10 +131,12 @@ impl InfluxDBSvc {
 impl Service<Vec<Metric>> for InfluxDBSvc {
     type Response = HttpResponse;
     type Error = HttpError;
-    type Future = Box<dyn Future<Item = Self::Response, Error = Self::Error> + Send + 'static>;
+    type Future = Pin<
+        Box<dyn std::future::Future<Output = Result<Self::Response, Self::Error>> + Send + 'static>,
+    >;
```

Comment:
Can be shortened with `BoxFuture` to:
`type Future = futures::future::BoxFuture<'static, Result<Self::Response, Self::Error>>`